src/app/mesh/qgsmeshlayerproperties.cpp

```diff
@@ -95,6 +95,11 @@ QgsMeshLayerProperties::QgsMeshLayerProperties( QgsMapLayer *lyr, QgsMapCanvas *
   delete mOptsPage_3DView;  // removes both the "3d view" list item and its page
 #endif
 
+  mComboBoxTemporalDatasetMatchingMethod->addItem( tr( "Find closest dataset before requested time" ),
```

Comment:
HIG convention is title case for combo box items (admittedly we aren't 100% on this, but are moving closer to it)